src/main/java/io/github/darealturtywurty/turtychemistry/core/MixinHooks.java

```diff
@@ -116,7 +116,7 @@ public static void calculateStackTemperature(final ItemStack stack, final Level
                     temperatureNBTTag = new CompoundTag();
                     temperatureNBTTag.putFloat(COMPOUND_TAG_ID, ThreadLocalRandom.current().nextFloat(28, 34));
                     stack.setTag(temperatureNBTTag);
-                } else if (temperatureNBTTag.getFloat(COMPOUND_TAG_ID) > 35f) {
+                } else if (temperatureNBTTag.getFloat(COMPOUND_TAG_ID) > 48f) {
```

Comment:
I'm not sure why this PR is changing this. And I also think 48c is too high. Either way, since you are messing around with it, it would be preferred if you made this value a constant, that (in the future) could be added to a config.